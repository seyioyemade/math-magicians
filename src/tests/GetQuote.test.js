import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import GetQuote from '../components/GetQuote';

describe('Get Quote form API', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<BrowserRouter><GetQuote /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
