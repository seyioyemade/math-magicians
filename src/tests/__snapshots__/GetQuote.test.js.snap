// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Get Quote form API renders correctly 1`] = `null`;
